packages/core/src/civil.ts

```diff
@@ -60,6 +61,10 @@ export class Civil {
     return this.ethApi.toBigNumber(num);
   }
 
+  public async signMessage(message: string, account?: EthAddress): Promise<EthSignedMessage> {
+    return this.ethApi.signMessage(message, account);
```

Comment:
Is this the correct method to use for signing the Constitution?

@ritave, I think you're the knowledgable party here

Reply:
Have we defined what "singing the Constitution" really means?
This function is correct in sense that it uses the currently open wallet to sign a message you provide with a cryptographic proof that it was this wallet.

But then who is the wallet in the first place? Is it one of the owners? Should they all sign it?

For now I'd say this is the best way until (if) we want to make a bigger ceremony out of signing the constitution